submissions/innazozulia/Hooli-style-popup/index.html

```diff
@@ -0,0 +1,120 @@
+
+                    <li class="submenu-item">
+                        <a href="#" class="submenu-link"><img src="img/calendar.png" alt="calendar-pic"></a>
+                        <div class='submenu-title'>Calendar</div>
+                    </li>                     
+                    <li class="submenu-item">
+                        <a href="#" class="submenu-link"><img src="img/contact-book.png" alt="contacts-pic"></a>
+                        <div class='submenu-title'>Contacts</div>
+                    </li> 
+                    <li class="submenu-item">
+                        <a href="#" class="submenu-link"><img src="img/search.png" alt="search-pic"></a>
+                        <div class='submenu-title'>Search</div>
+                    </li>                    
+                    <li class="submenu-item">
+                        <a href="#" class="submenu-link"><img src="img/chat.png" alt="chat-pic"></a>
+                        <div class="submenu-title">Chat</div>
+                    </li>
+                    <li class="submenu-item">
+                        <a href="#" class="submenu-link"><img src="img/newspaper.png" alt="news-pic"></a>
+                        <div class='submenu-title'>News</div>
+                    </li> 
+                    <li class="submenu-item">
+                        <a href="#" class="submenu-link"><img src="img/mail.png" alt="mail-pic"></a>
+                        <div class='submenu-title'>Mail</div>
+                    </li> 
+                    <li class="submenu-item">
+                        <a href="#" class="submenu-link"><img src="img/file.png" alt="files-pic"></a>
+                        <div class='submenu-title'>Files</div>
+                    </li> 
+                    <li class="submenu-item">
+                        <a href="#" class="submenu-link"><img src="img/translation.png" alt="translate-pic"></a>
+                        <div class='submenu-title'>Translate</div>
+                    </li> 
+                    <li class="submenu-item">
+                        <a href="#" class="submenu-link"><img src="img/youtube.png" alt="videos-pic"></a>
+                        <div class='submenu-title'>Videos</div>
+                    </li> 
+
+                    <!-- More bottom -->
+
+                    <div class="submenu-bottom">
+                        <input type="checkbox" id="more-checkbox" class="more-checkbox hidden-checkbox">
+                                <label for="more-checkbox" class='more'>More</label>
+                                <hr>
+                                <div class="submenu-hidden">
+                                    <ul class="submenu-list">
+                                        <li class="submenu-item">
+                                            <a href="#" class="submenu-link"><img src="img/map.png" alt="map-pic"></a>
+                                            <div class='submenu-title'>Map</div>
```

Comment:
Pay attention please to the semantics of your layout. Try to use another tags fro this submenu item. I see that you use some text item here. So maybe `span`, `p` or title tags would be better here